packages/ma-components/local/page-footer/page-footer.css

```
@@ -0,0 +1,99 @@
+.ma-page-footer {
+  --nl-paragraph-color: currentColor;
+  --nl-heading-level-4-color: currentColor;
+  --nl-link-color: currentColor;
+  --nl-link-text-decoration-color: currentColor;
+  --nl-link-hover-color: var(--ma-page-footer-color);
+
+  background-color: var(--ma-page-footer-background-color);
+  box-sizing: border-box;
+  color: var(--ma-page-footer-color);
+  container-name: page-footer;
```

Als je consistent wil zijn met `ma-page-header` in de andere PR:

```suggestion
  container-name: ma-page-footer;
```